cli/project/dockerfiles.go

```diff
@@ -187,29 +187,42 @@ const (
 
 	containerSDKPath = "/usr/share/tarantool/sdk"
 
-	defaultBaseLayers          = "FROM centos:7\n"
+	defaultBaseLayers = "FROM centos:7\n"
+
 	installBuildPackagesLayers = `### Install packages required for build
 RUN yum install -y git-core gcc gcc-c++ make cmake unzip
+`
+	// We have to set USER instruction in the form of <UID:GID>
+	// (see https://github.com/tarantool/cartridge-cli/issues/481).
+	// Since we cannot find out the already set UID and GID for the tarantool
+	// user using command shell (see https://github.com/moby/moby/issues/29110),
+	// we recreate the user and the tarantool group with a constant UID and GID value.
+
+	createTarantoolUser = `RUN groupadd -r -g {{ .TarantoolGID }} tarantool \
```

Comment:
Let's start this part with a one-line comment (like other parts).

Smth like `# Create Tarantool user`